packages/ui/cypress/tests/multisig-creation.cy.ts

```diff
@@ -144,14 +145,12 @@ describe('Multisig creation', () => {
 
       // The banner should be there, and disapear within 30s
       landingPage.multisigCreationInfoBanner().should('be.visible')
+      landingPage.creationInfoBannerCloseButton().click()
 
-      cy.clock().then((clock) => {
```

Comment:
this wasn't working. Adding the cy.clock before the cy.visit actually breaks everything, somehow, as the api calls don't get fired. All in all removing the clock